tsd/tsd.go

```diff
@@ -48,7 +48,7 @@ type System struct {
 	Tun            SubSystem[*tstun.Wrapper]
 	StateStore     SubSystem[ipn.StateStore]
 	Netstack       SubSystem[NetstackImpl] // actually a *netstack.Impl
-	TailfsForLocal SubSystem[*tailfs.FileSystemForLocal]
+	TailFSForLocal SubSystem[*tailfs.FileSystemForLocal]
```

Comment:
this adds 1MB to tsnet applications that don't use tailfs, sadly.

Can we use an interface here instead of a concrete type?

Reply:
Fixed.